client/my-sites/email/form/mailboxes/components/style.scss

```diff
@@ -0,0 +1,92 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.list__main {
+	.form-fieldset {
+		.form-label {
+			margin-bottom: 0;
+
+			.form-text-input-with-affixes__suffix,
+			.form-text-input-with-affixes__prefix {
+				font-weight: normal;
+			}
+		}
+
+		.form-label > input[type=text],
+		.form-label > .form-text-input-with-affixes,
+		.form-label > .form-password-input {
```

Comment:
Sorry for bombarding in here even if you haven't marked this PR as ready for review in Asana, but I have some thoughts and in the interest of saving time down the road, I thought I'd give my 2 cents here now 😊

The Calypso [CSS guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md) don't explicitly mention BEM, but still, that is the general pattern we're following for class names. Both BEM and the CSS guidelines [discourage](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#nesting) the use of nested selectors, so I think it might be worthwhile to refactor this CSS a little bit.

A few more concrete suggestions below. If these are hindered by competing specificity from CSS selectors elsewhere, I'd be happy to register a follow up ticket to try and reduce specificity in all necessary places.

1. All of the `form-` classes stem from those components and I do realize that it's fairly common practice across Calypso to use nested selectors to target those. Still, we could either use more shallowly nested CSS selectors or we could pass complementary class names to those components (e.g. `mailboxes-list__form-label`).
2. `list__main` is quite generic. We could use something like `new-mailbox-list__main`.
3. Same for `list__actions`, `list__supplied-actions` and `list__main-actions`. It could be e.g. `new-mailbox-list__actions`. It also probably doesn't have to be nested inside `list__main`.